worlds/sc2/mission_order/structs.py

```diff
@@ -326,15 +326,15 @@ def fill_min_steps(self):
                 cur_layouts[campaign.option_name] = set()
         next_campaigns.difference_update(cur_campaigns)
         next_campaigns.update(new_campaigns)
-        for campaign in next_layouts:
+        for campaign_name, layouts in next_layouts.items():
             new_layouts: Set[SC2MOGenLayout] = set()
```

Comment:
mypy still complains about this declaration and the one below it for re-declaring a variable / type here. I think this is fine; it may be good to rename the variables / remove the type annotation if it matches the usage above, but I'll hold off on that until it's on sc2-next and I feel like doing another mypy issue bugquashing PR.